blocks/donation-form-grid/edit/inspector.js

```diff
@@ -98,7 +98,20 @@ const Inspector = ( { attributes, setAttributes } ) => {
 					label={ __( 'Show Excerpt', 'give' ) }
 					checked={ !! showExcerpt }
 					onChange={ ( value ) => saveSetting( 'showExcerpt', value ) } />
-				<ToggleControl
+                <TextControl
+                    name="excerptLength"
+                    label={ __( 'Excerpt Length', 'give' ) }
+                    value={ excerptLength ? excerptLength : '' }
+                    className={ !showExcerpt && 'hidden' }
+                    onChange={ ( value ) => {
```

Comment:
We are setting the `excerptLength` value only if `value === ''` and `intValue == value`?
Users should be able to set `excerptLength` to some arbitrary value, but they can't do that now.

Reply:
Resolved 68ecac6d

Reply:
Users probably shouldn't be able to set `excerptLength` to any arbitrary value, only an empty string or a positive integer. With the way it's just been refactored, the user can set a negative `excerptLength`. Not the biggest deal, and I appreciate the simplification of the code, just something to consider.